modules/openapi-generator/src/main/resources/kotlin-client/libraries/jvm-okhttp/infrastructure/ApiClient.kt.mustache

```diff
@@ -209,10 +210,10 @@ import com.squareup.moshi.adapter
                         .toRequestBody((mediaType ?: JsonMediaType).toMediaTypeOrNull())
                 }
             mediaType == XmlMediaType -> throw UnsupportedOperationException("xml not currently supported.")
-            mediaType == OctetMediaType && content is ByteArray ->
-                content.toRequestBody(OctetMediaType.toMediaTypeOrNull())
+            mediaType == TextMediaType && content is String ->
+                content.toRequestBody(TextMediaType.toMediaTypeOrNull())
```

Comment:
@Djaler did you deleted OctetMediaType by accident or on propose?

Reply:
On purpose. See message for this commit https://github.com/OpenAPITools/openapi-generator/commit/c5836ffa9885bd05b640dc6ea038e43aa6318947